site/content/docs/skills/migrate.md

```
@@ -66,7 +79,24 @@ Harvest a DCAT catalog (DKAN / ArcGIS Hub / data.gov):
 /migrate https://hub.arcgis.com/data.json --source dcat
 ```
 
-## Copy modes
+Harvest a Socrata site, an OpenDataSoft portal, or a single ArcGIS service:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)